tests/Linter/TokenizerLintingResultTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Linter;
+
+use PhpCsFixer\Linter\TokenizerLintingResult;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Linter\TokenizerLintingResult
+ */
+final class TokenizerLintingResultTest extends TestCase
+{
+    /**
+     * @doesNotPerformAssertions
+     */
+    public function testTokenizerLintingResultOK()
+    {
+        $result = new TokenizerLintingResult();
+        $result->check();
+
+        $this->addToAssertionCount(1);
```

Comment:
is it still needed when we do have annotation now ?

Reply:
no, not sure why I only removed the comment :|